### PR TITLE
i18n(es): add buddy deploy

### PR DESCRIPTION
**Pull request by kevinzunigacuellar**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- Adds `buddy.md` in spanish

### CODE REVIEW

File: src/pages/es/guides/deploy/buddy.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+title: Despliega tu proyecto de Astro con Buddy
+description: Cómo desplegar tu proyecto Astro usando Buddy.
+layout: ~/layouts/DeployGuideLayout.astro
+i18nReady: true
+---
+
+Puedes desplegar tu proyecto Astro usando [Buddy](https://buddy.works/), una solución de CI/CD que puede construir tu proyecto y enviarlo a muchos destinos de despliegue diferentes, incluidos servidores FTP y proveedores de alojamiento en la nube.
+
+:::note
+Buddy no alojará tu proyecto. Solo lo ayudará a administrar el proceso de compilación y entregará el resultado a la plataforma de despliegue de tu elección.
```
Comment (agustinmulet) [inline src/pages/es/guides/deploy/buddy.md:11]:
```suggestion
Buddy no alojará tu proyecto. Solo te ayudará a administrar el proceso de compilación y entregará el resultado a la plataforma de despliegue de tu elección.
```

File: src/pages/es/guides/deploy/buddy.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+:::
+
+## Cómo desplegar
+
+1. Cree una cuenta en [Buddy](https://buddy.works/sign-up).
+2. Cree un nuevo proyecto y conéctelo con un repositorio Git (GitHub, GitLab, BitBucket, cualquier repositorio Git privado o puede usar Buddy Git Hosting).
+3. Agregue una nueva pipeline.
```
Comment (agustinmulet) [inline src/pages/es/guides/deploy/buddy.md:18]:
```suggestion
3. Agrega una nueva pipeline.
```

File: src/pages/es/guides/deploy/buddy.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+4. En la pipeline recién creada, agregue una action para **[Node.js](https://buddy.works/actions/node-js)**.
```
Comment (agustinmulet) [inline src/pages/es/guides/deploy/buddy.md:19]:
```suggestion
4. En la pipeline recién creada, agrega una action para **[Node.js](https://buddy.works/actions/node-js)**.
```

File: src/pages/es/guides/deploy/buddy.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+5. En esta acción agregue:
```
Comment (agustinmulet) [inline src/pages/es/guides/deploy/buddy.md:20]:
```suggestion
5. En esta action agrega:
```

File: src/pages/es/guides/deploy/buddy.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+   ```bash
+   npm install
+   npm run build
+   ```
+
+6. Agregue una action de despliegue: hay muchas para elegir, puedes explorarlas [aquí](https://buddy.works/actions). Aunque tus configuraciones pueden diferir, recuerde configurar la **ruta de origen** en `dist`.
```
Comment (agustinmulet) [inline src/pages/es/guides/deploy/buddy.md:27]:
```suggestion
6. Agrega una action de despliegue: hay muchas para elegir, puedes explorarlas [aquí](https://buddy.works/actions). Aunque tus configuraciones pueden diferir, recuerda configurar la **ruta de origen** en `dist`.
```

File: src/pages/es/guides/deploy/buddy.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+7. Presione el botón **Run**.
```
Comment (agustinmulet) [inline src/pages/es/guides/deploy/buddy.md:28]:
```suggestion
7. Presiona el botón **Run**.

```

**Comment by Yan-Thomas** [comment]
My favorite time of the day: update and merge amazing contributions! :partying_face: Thank you yet again for this amazing job @kevinzunigacuellar, @agustinmulet :purple_heart:

**Pull request merged**